### DISCUSSION TITLE
Pull shark check

### DISCUSSION THREAD

**Discussion by goatganyu** [community/community#19823] [Other Feature Feedback, Questions, & Ideas]
Pull shark: How to track the streak? I wonder, how to check my total PRs count that have been merged?

**Comment by sinkaroid** [comment]
I guess the https://api.github.com should handle that,  
Also you can using profile workflows like https://github.com/lowlighter/metrics, and generate your profile with [Follow-up of issues and pull requests](https://github.com/lowlighter/metrics/blob/master/source/plugins/followup/README.md) plugins You will see how many PRs from your account that have been merged. 
[![](https://i.imgur.com/PlYgTr6.png)](https://i.imgur.com/PlYgTr6.png)

**Reply by goatganyu** [reply]
Thanks, seems the github api docs does not tell something about how to get PR count..
Solved by using those profile templates
